radicle-cli/src/commands/issue.rs

```diff
@@ -235,7 +235,22 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
         Operation::List => {
             for result in issues.all()? {
                 let (id, issue, _) = result?;
-                println!("{} {}", id, issue.title());
+
+                let assigned: String = issue
+                    .assigned()
+                    .map(|p| p.to_string())
+                    .collect::<Vec<_>>()
+                    .join(", ");
+                if assigned.is_empty() {
+                    println!("{} \"{}\"", id, issue.title().escape_default());
+                } else {
+                    println!(
```

Comment:
Actually, how come you're using `println` here instead of the terminal code?

Reply:
Yea probably a good idea. I'd also prefer if we used `{:?}` for printing a quoted string, as escape sequences are hard to read.

Reply:
Is there a better way to print quoted strings without operating through debug output?  Escaping quotes may be better for reading, but using '{:?}' is using debug code out of the context of debugging.

Reply:
> Actually, how come you're using `println` here instead of the terminal code?

Using the terminal crate's Table formatter now.

Reply:
> Is there a better way to print quoted strings without operating through debug output? Avoiding escaping quotes may be better for reading, but using '{:?}' is using debug code out of the context of debugging.

Hmm not really. We probably won't be using quotes anyway once we design this properly, so it's not something to worry about long term. If you prefer, you can simply print it unquoted for now.

Reply:
I've given this more thought, and I think that if we want quoted strings, using `{:?}` is the right solution, because:
1. It does exactly that: it quotes the string and escapes any `"` characters
2. Given that it's part of `String`'s public API, we can rely on it not breaking
3. `{:?}` is a standard formatting option and `std::fmt::Debug` is a standard trait. Just because it's called `Debug` doesn't mean it should not be used in production. In this case, the debug output is exactly what we're looking for (if we choose to display quoted strings).

Reply:
Ok, I'll use this more consistently moving forwards.